src/Uno.UWP/ApplicationModel/Resources/ResourceLoader.cs

```diff
@@ -184,6 +184,17 @@ private static IEnumerable<string> GetCulturesHierarchy(CultureInfo culture)
 			{
 				yield return culture.IetfLanguageTag.ToLowerInvariant();
 
+				// If we have a culture that doesn't specify country/region, we want to match its specific cultures if it's not found.
+				// For example, if we have es and it's not found, we want to match es-MX
+				if (culture.IsNeutralCulture)
+				{
+					var specificCultures = CultureInfo.GetCultures(CultureTypes.SpecificCultures).Where(c => c.Parent.Equals(culture));
```

Comment:
Very nice! WinUI seems to be ordering the cultures alphabetically; we should do the same so that es-ES is used instead of es-MX. Make sure to revalidate this is really the case on Windows, though.

Reply:
@jeromelaban In UWP, `es-MX` is used, not `es-ES`. It matches the last culture sorted alphabetically.

`GetCultures` already returns cultures sorted. The thing is that I was expecting my code to pick `es-ES` (which is wrong). But actually the code did the correct thing and picked `es-MX` matching UWP.

I don't know why, but it seems to work just like UWP 😄

Reply:
Ok then, let's keep it this way then! Thanks :)